apps/search/js/providers/suggestions.js

```diff
@@ -44,8 +52,6 @@
 
     name: 'Suggestions',
 
-    remote: true,
```

Comment:
Why was this removed? I think this is definitely still needed for private browsing cases.